sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConsumerSpEL.java

```diff
@@ -90,6 +98,16 @@ public ConsumerSpEL() {
     } catch (NoSuchMethodException | SecurityException e) {
       LOG.debug("OffsetsForTimes is not available.");
     }
+
+    try {
+      // It is supported by Kafka Client 2.1.0 onwards.
+      Method method =
+          Deserializer.class.getDeclaredMethod(
+              "deserialize", String.class, Headers.class, byte[].class);
+      deserializerSupportsHeaders = method.isDefault();
```

Comment:
Why does this matter that it is a default method?
Shouldn't we be setting this to true as long as the method exists?